docs/tutorial.rst

```diff
@@ -207,6 +207,51 @@ Passing options to select2
 
 .. note:: Setting a placeholder will result in generation of an an empty
           ``option`` tag, which select2 requires.
+          
+Using autocomplete with URL parameters
+======================================
+
+DAL allows the use of the Django `redirect
+<https://docs.djangoproject.com/en/4.0/topics/http/shortcuts/#redirect>`_
```

Comment:
The example does not use that redirect function, did you mean reverse() ?

Reply:
Yes. Sorry, I mistyped that haha!